testsuites/syncgateway/performance/kill_gateload.py

```diff
@@ -1,10 +1,22 @@
 import os
-import subprocess
+import sys
+from libraries.provision.ansible_runner import AnsibleRunner
 
 
 def kill_gateload():
-    os.chdir("performance_tests/ansible/playbooks")
-    subprocess.call(["ansible-playbook", "-i", "../../../provisioning_config", "kill-gateload.yml"])
+    try:
+        cluster_config = os.environ["CLUSTER_CONFIG"]
+    except KeyError:
+        print ("Make sure CLUSTER_CONFIG is defined and pointing to the configuration you would like to provision")
+        sys.exit(1)
+
+    print("Running perf test against cluster: {}".format(cluster_config))
+    ansible_runner = AnsibleRunner(cluster_config)
+
+    status = ansible_runner.run_ansible_playbook("kill-gateload.yml")
+    if status != 0:
+        print("Killing gateload returned non-zero status: {}.  Most likely it was no longer running".format(status))
```

Comment:
> In what cases will gateload not be finished at this point?  Why do we have to kill gateload? I thought is ended on its own after the runtime_ms.

If sync gateway crashes (due to OOM or otherwise), the `log_expvars.py` script will exit because it can no longer reach Sync Gateway.

However, gateload might still be running, which is not really desirable, especially since sync gateway automatically restarts by default (which I wish I could disable for the tests) 

So what ends up happening:

1. `log_expvars.py` polls sync gateway and gateload -- everything OK
1. Sync Gateway crashes and is unresponsive while it panics and dumps 10s of thousands of goroutines
1. `log_expars.py` tries to reach sync gateway, and it can't.  The script ends and returns `successful=false`
1. Sync Gateway restarts
1. Jenkins job ends
1. Gateload continues hitting restarted sync gateway and stats continue to be pushed to grafana <-- confusing